packages/fonts/src/index.css

```
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: 'Daimler CS';
+  font-style: normal;
+  font-weight: 350;
+  src: url('./DaimlerCS-Regular.woff') format('woff'),
+    url('./DaimlerCS-Regular.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'Daimler CS';
+  font-style: normal;
+  font-weight: 700;
+  src: url('./DaimlerCS-Bold.woff') format('woff'),
+    url('./DaimlerCS-Bold.woff2') format('woff2');
+}
```

font-weight should be `400` for the regular font file.
change order of sources (prefer woff2 for browser download)

```suggestion
@font-face {
  font-family: 'Daimler CS';
  font-style: normal;
  font-weight: 400;
  src: url('./DaimlerCS-Regular.woff2') format('woff2'),
    url('./DaimlerCS-Regular.woff') format('woff');
}

@font-face {
  font-family: 'Daimler CS';
  font-style: normal;
  font-weight: 700;
  src: url('./DaimlerCS-Bold.woff2') format('woff2'),
    url('./DaimlerCS-Bold.woff') format('woff');
}
```